### PR TITLE
Adds getVersion & getFirstAvailableBlock method

### PR DESCRIPTION
**Pull request by OmkarAcharekar**
The pr adds 2 json rpc requests  of issue #39
* [getVersion](https://solana-labs.github.io/solana-web3.js/classes/Connection.html#getVersion) 
* [getFirstAvailableBlock](https://solana-labs.github.io/solana-web3.js/classes/Connection.html#getFirstAvailableBlock)

### CODE REVIEW

**Review by aniketfuryrocks — APPROVED** [review]

**Comment by aniketfuryrocks** [comment]
Waiting for #61

**Comment by aniketfuryrocks** [comment]
@OmkarAcharekar #61 Got merged. Please pull it and update the pr.

File: tests/main.cpp [hunk 1]
```diff
@@ -707,3 +707,19 @@ TEST_CASE("account9") {
...
+  // get Version
+  const auto version = connection.getVersion();
+  CHECK_GT(version.solana_core.length(), 0);
+}
+
+TEST_CASE("getFirstAvailableBlock") {
+  const solana::Keypair keyPair = solana::Keypair::fromFile(KEY_PAIR_FILE);
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  // get Version
+  const auto firstAvailableBlock = connection.getFirstAvailableBlock();
```
Comment (godmodegalactus) [inline tests/main.cpp:723]: Can also add a test like
```
const auto slot = connection.getSlot(); // get finalized slot.
const auto firstAvailableBlock = connection.getFirstAvailableBlock();
CHECK_GT(firstAvailableBlock, slot);
sleep(2); // sleep for 2 seconds
const auto availableBlock = connection.getFirstAvailableBlock();
CHECK_GT(availableBlock, firstAvailableBlock);
```

**Review by godmodegalactus — CHANGES_REQUESTED** [review]

File: tests/main.cpp (hunk 1 above)
Comment (godmodegalactus) [reply tests/main.cpp:723]: Yes this does not really work because getAvailableBlock is the block in the ledger. I guess you can keep it like it is.

Comment (OmkarAcharekar) [reply tests/main.cpp:723]: Ok

File: tests/main.cpp [hunk 2]
```diff
@@ -707,3 +707,19 @@ TEST_CASE("account9") {
   CHECK_EQ(leverage.to_double(), 3.919442838288937);
   CHECK_FALSE(mangoAccount.isLiquidatable(mangoGroup, mangoCache));
 }
+
+TEST_CASE("getVersion") {
+  const solana::Keypair keyPair = solana::Keypair::fromFile(KEY_PAIR_FILE);
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  // get Version
+  const auto version = connection.getVersion();
+  CHECK_GT(version.solana_core.length(), 0);
```
[the remark this answers is not in the document]
Comment (OmkarAcharekar) [reply tests/main.cpp]: @godmodegalactus Done the required changes ! Thanks

File: tests/main.cpp [hunk 3]
```diff
@@ -707,3 +708,19 @@ TEST_CASE("account9") {
   CHECK_EQ(leverage.to_double(), 3.919442838288937);
   CHECK_FALSE(mangoAccount.isLiquidatable(mangoGroup, mangoCache));
 }
+
+TEST_CASE("getVersion") {
+  const solana::Keypair keyPair = solana::Keypair::fromFile(KEY_PAIR_FILE);
+  const auto connection = solana::rpc::Connection(solana::DEVNET);
+  const auto version = connection.getVersion();
+  boost::regex expression{"\\d+.\\d+.\\d+"};
```
Comment (godmodegalactus) [inline tests/main.cpp:716]: Hey previously I missed '\.', just '.' will match any character. otherwise PR looks good.

**Review by godmodegalactus — APPROVED** [review]

**Pull request merged**